### PR TITLE
ALL-2878 - Add Beacon methods to eth

### PR DESCRIPTION
**Pull request by Hathoriel**
adds #998

### CODE REVIEW

File: src/util/util.shared.ts [hunk 1]
```diff
@@ -547,4 +552,28 @@ export const Utils = {
     }
     return rpc?.nodes?.[0].url || `${Constant.TATUM_API_URL.V3}blockchain/node/${network}`.concat(path || '')
   },
+  addQueryParams: (basePath: string, queryParams?: Record<string, string | string[]>): string => {
```
Comment (hehe100596) [inline src/util/util.shared.ts:555]: Formatting in this file seems off, lines are not aligned / indented correctly.

File: src/service/rpc/generic/LoadBalancer.ts [hunk 2]
```diff
@@ -42,6 +43,20 @@ interface InitRemoteHostsParams {
   noSSRFCheck?: boolean
 }
 
+enum RequestType {
```
Comment (Smrecz) [inline src/service/rpc/generic/LoadBalancer.ts:46]: don't we have other types? like DELETE for unsubscribe?

File: src/util/util.shared.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { version } from '../../package.json'
 import {
   AddressEventNotificationChain,
   isEosLoadBalancerNetwork,
-  isEosNetwork,
+  isEosNetwork, isEvmArchiveNonArchiveBeaconLoadBalancerNetwork,
```
Comment (Smrecz) [inline src/util/util.shared.ts:8]: lint me pls

File: src/service/rpc/generic/LoadBalancer.ts (hunk 2 above)
Comment (Hathoriel) [reply src/service/rpc/generic/LoadBalancer.ts:46]: I dont understand what do you mean - this is different use case. It is only for purpose of logging.

File: src/util/util.shared.ts (hunk 3 above)
Comment (Hathoriel) [reply src/util/util.shared.ts:8]: fixed

File: src/util/util.shared.ts (hunk 1 above)
Comment (Hathoriel) [reply src/util/util.shared.ts:555]: image

**Review by Smrecz — APPROVED** [review]

File: src/service/rpc/generic/LoadBalancer.ts (hunk 2 above)
Comment (Smrecz) [reply src/service/rpc/generic/LoadBalancer.ts:46]: nvm, thought it's on a different level

**Pull request merged**